compiler/runtime/OMRCodeCache.cpp

```diff
@@ -421,6 +421,13 @@ OMR::CodeCache::initialize(TR::CodeCacheManager *manager,
    *((TR::CodeCache **)(_segment->segmentBase())) = self();
    omrthread_jit_write_protect_enable();
 
+   if (OMR::RSSReport::instance())
+      {
+      // Down-stream project should set page size
```

Comment:
I'm not clear why the downstream project has to set the page size.  Does the `OMR::CodeCache` not know (or could find out) the page size for the code cache memory it is representing?

Reply:
I think down-stream project can have it's own port library and has its own way of finding the page size. So I thought it's better to leave it to a specific project.

Reply:
I think it's OK to leave that comment here but also document in RSSReport.hpp

Reply:
Reworked.